common/src/main/java/com/company/common/models/UserEntity.java

```diff
@@ -12,6 +15,8 @@
 public class UserEntity {
 
     @Id
+    @GeneratedValue
+    @JdbcType(VarcharJdbcType.class)
```

Comment:
@Id
@GeneratedValue(strategy = GenerationType.UUID)

@JdbcType(VarcharJdbcType.class)do w need this?

Reply:
Repository did not work with search by id for uuid

Reply:
i don't think so
https://www.baeldung.com/java-hibernate-uuid-primary-key

are you getting error?

Reply:
Yes, in another PR where I did the registration, I got an error

Reply:
I changed, now everything works, I checked